trunk/src/index.js

```diff
@@ -0,0 +1,186 @@
+	 * This is the display title for your block, which can be translated with `i18n` functions.
+	 * The block inserter will show this name.
+	 */
+	title: __('Pardot Form', 'pardot'),
+
+	/**
+	 * This is a short description for your block, can be translated with `i18n` functions.
+	 * It will be shown in the Block Tab in the Settings Sidebar.
+	 */
+	description: __(
+		'Collect information about people visiting your site or landing page and help you turn anonymous visitors into identified prospects.',
+		'pardot'
+	),
+
+	/**
+	 * Blocks are grouped into categories to help users browse and discover them.
+	 * The categories provided by core are `common`, `embed`, `formatting`, `layout` and `widgets`.
+	 */
+	category: 'embed',
+
+	/**
+	 * An icon property should be specified to make it easier to identify a block.
+	 * These can be any of WordPress’ Dashicons, or a custom svg element.
+	 */
+	icon: icon,
+
+	/**
+	 * Optional block extended support features.
+	 */
+	supports: {
+		// Removes support for an HTML mode.
+		html: false,
+	},
+
+	attributes: {
+		form_id: {
+			type: 'string',
+			default: '',
+		},
+		height: {
+			type: 'string',
+			default: '',
+		},
+		width:{
+			type: 'string',
+			default: '',
+		},
+		className:{
+			type: 'string',
+			default: '',
+		},
+		title:{
+			type: 'string',
+			default: '',
+		},
+	},
+
+	/**
+	 * @see ./edit.js
+	 */
+	edit: form_edit,
+
+	save: function(props) {
+		return null;
+	}
+});
+
+/**
+ * Every block starts by registering a new block type definition.
+ *
+ * @see https://developer.wordpress.org/block-editor/developers/block-api/#registering-a-block
+ */
+registerBlockType('pardot/dynamic-content', {
+	/**
+	 * This is the display title for your block, which can be translated with `i18n` functions.
+	 * The block inserter will show this name.
+	 */
+	title: __('Pardot Dynamic Content', 'pardot'),
+
+	/**
+	 * This is a short description for your block, can be translated with `i18n` functions.
+	 * It will be shown in the Block Tab in the Settings Sidebar.
+	 */
+	description: __(
+		'Delivers targeted messaging to prospects. Content is displayed according to rules defined in Pardot based on the prospect’s data or attributes.',
+		'pardot'
+	),
+
+	/**
+	 * Blocks are grouped into categories to help users browse and discover them.
+	 * The categories provided by core are `common`, `embed`, `formatting`, `layout` and `widgets`.
+	 */
+	category: 'embed',
+
+	/**
+	 * An icon property should be specified to make it easier to identify a block.
+	 * These can be any of WordPress’ Dashicons, or a custom svg element.
+	 */
+	icon: icon,
+
+	/**
+	 * Optional block extended support features.
+	 */
+	supports: {
+		// Removes support for an HTML mode.
+		html: false,
+	},
+
+	attributes: {
+		dynamicContent_id: {
+			type: 'string',
+			default: '',
+		},
+		dynamicContent_default: {
+			type: 'string',
+			default: '',
+		},
+		height: {
+			type: 'string',
+			default: '',
+		},
+		width:{
```

Comment:
yeah I wonder why this isn't defined as a number. What's the advantage of setting this to text/string

Reply:
also, is there a logical default we can set?